docker-compose.yml

```
@@ -3,7 +3,7 @@ version: '2.3'
 services:
   statping:
     container_name: statping
-    image: aboutcher/statping-ng:dev
+    image: statping-ng/statping-ng:dev
```

that's not right, but it's not your fault ;) https://hub.docker.com/r/statping-ng/statping-ng doesn't exist, but we should probably switch to that at some point ^^
```suggestion
    image: aboutcher/statping-ng:dev
```

---

that's where the problems start :D by using aboutcher/statping the image will always be pulled directly from dockerhub. or we need to build every image as aboutcher/statping locally. But for now we can keep it. I guess

---

ooh I see. for now, there's kind of a "split workflow":
* if you want to deploy statping somewhere, you use the aboutcher image -> pulled from docker hub
* if you want to work on statping, you use the statping-ng image in combination with the compose files in dev/ -> built locally